pkg/cache/cache.go

```diff
@@ -0,0 +1,258 @@
+package cache
+
+import (
+	glog "github.com/Sirupsen/logrus"
```

Comment:
Let's not alias logrus as glog - I think that's confusing since it's not glog.

Please just call it `log`